src/Microsoft.AspNet.Mvc.TagHelpers/FormTagHelperOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Mvc.TagHelpers
+{
+    /// <summary>
+    /// Options pertaining to the default behavior of <see cref="FormTagHelper"/> instances.
+    /// </summary>
+    public class FormTagHelperOptions
+    {
+        /// <summary>
+        /// Whether the anti-forgery token should be generated by default for all instances of
+        /// <see cref="FormTagHelper"/>. Can be overridden on any given instance.
+        /// </summary>
+        /// <value>
+        ///     Defaults to <c>null</c>, which indicates a token will only be generated if the <c>action</c>
+        ///     attribute was not explicitly defined.
+        /// </value>
+        public bool? AntiForgery { get; set; }
```

Comment:
Thinking about it more, I'm not sure it matters so much that the properties on the options object exactly match their counterparts. We could make the options property `public bool IncludeAntiForgeryToken { get; set; }` (dropping the nullable bool) and update the logic accordingly.